safe-room.lic

```diff
@@ -72,6 +72,9 @@ class SafeRoom
   def wait_for_healing
     Flags.reset('healthy')
     Flags.reset('moved')
+    
+    lie_down
+    wait_for_script_to_complete('tendme')
```

Comment:
Does `tendme` ever actually end? The code looks like a loop that just repeatedly tends and never exits.